frontend/packages/core/src/accordion.tsx

```diff
@@ -89,7 +85,7 @@ const StyledAccordionGroup = styled.div({
     marginBottom: "16px",
   },
 
-  ".MuiAccordion-root:before": {
+  "&.MuiAccordion-root:before": {
```

Comment:
not sure this is correct, do you remember why you made this change? also since this is a styled div the root class will never have the root class i think

originally the class was to remove a top divider element that accordiongroup children `[1:]` get

Reply:
Ah I can drop this since I don't think it worked anyway. Right now we have an issue on the second step of the workflow where the before is still rendering if the accordion is a child. Need to figure out a fix for that but it can happen in a subsequent PR